### ISSUE TITLE
[Pytorch] Visualization graph

### ISSUE THREAD

**Issue by Nakkwan** [Nakkwan/Nakkwan.github.io#1]
pytorch에서 Loss나 Accuracy 등의 시각화는 visdom을 통해서 할 수 있다. visdom의 설치는  `pip install visdom` <br>
visdom의 서버 실행은 `python -m visdom.server`로 실행 시킬 수 있고, <br>
`ip주소:8907`을 입력하면, ex)`192.168.0.0:8907` visdom 서버 창이 뜨게 된다. <br>
```py
import visdom
import numpy as np
import torch

vis = visdom.Visdom()
plot = vis.line(Y = torch.randn(5), X = np.array([0,1,2,3,4]))
```
의 형식으로 쓸 수 있다.<br>
for문 안을 도는 visdom plot의 경우
```py
import visdom
import numpy as np
import torch

vis = visdom.Visdom()
plot = vis.line(Y = torch.randn(5), X = np.array([0,1,2,3,4]))

for i n range(100):
    vis.line(Y = [random.random()], X = np.array([i+5]), win=plot, update='append')
```
의 형식으로 구현할 수 있다.

### Visdom Line
visdom에서 line은 
```py
plot = vis.line(Y=torch.zeros(1), X = torch.Tensor([0]), opts=dict(xlabel='i', ylabel='y', title='title', legend=['line1']))
```
의 형식으로 나타낼 수 있다. 추가적이 option은 opts에서 설정이 가능하다.

multiple line의 경우, 
```py
plot = vis.line(Y=torch.rand(10, n), X = torch.Tensor(range(0, 10)), opts=dict(xlabel='i', ylabel='y', title='title', legend=['line1', 'line2', 'linen']))
```
과 같이 n개의 line을 그릴 수 있으며, for문 안에 지속적으로 값을 추가하고 싶을 시, 
```py
vis.line(Y=torch.Tensor([constantA]), X = torch.Tensor([i]), win='plot', update='append', name='line1')
```
과 같이 legend에 적힌 line의 이름을 name 옵션으로 설정해주면 된다.
